learn/data_backup/snapshots.mdx

```diff
@@ -1,86 +1,56 @@
+To import snapshot data into your instance, launch Meilisearch using `--import-snapshot`:
 
 ```bash
 meilisearch --import-snapshot mySnapShots/data.ms.snapshot
 ```
 
-## Common problems
-
-Take note that whenever you launch Meilisearch from a snapshot, it will **stop processing and throw an error** if it encounters either of the two following situations:
-
-1. A database already exists, meaning you have a non-empty `data.ms` folder in the same directory as your Meilisearch binary
-2. No snapshot is found at the given path
-
-In both cases, **this behavior is [configurable](/learn/configuration/instance_options#ignore-missing-snapshot)**.
-
-If you don't want Meilisearch to throw an error when finding that a database already exists, you can add the following flag: `--ignore-snapshot-if-db-exists=true`. When using this flag, Meilisearch will use the existing database to start an instance instead of throwing an error. The snapshot will be ignored.
-
-If you do not want Meilisearch to throw an error when there is no snapshot at the given path, you can add the following flag: `--ignore-missing-snapshot`. Meilisearch will then continue its process and not import any snapshot.
-
-When starting from a snapshot, chances are that you already have an existing database. **For security reasons, a database is never overwritten**. To load a snapshot when an existing database is present, you will have to manually delete the existing database. By default, this is the contents of the `data.ms` folder (unless you [changed the path](/learn/configuration/instance_options#database-path)) which is located in the same folder as your Meilisearch binary.
-The simplest way to delete your database is with the terminal command `rm -rf data.ms`, after which you should be able to start Meilisearch with a snapshot.
-
-[[More about snapshots flags and environment variables]](/learn/configuration/instance_options#schedule-snapshot-creation)
-
-## Use cases
-
-**Snapshots are safeguards in case of problems**. If your Meilisearch instance encounters a problem or if you make a mistake while manipulating your database, restarting your instance with the latest snapshot is an easy way to recover your data.
+Because snapshots are exact copies of your database, starting a Meilisearch instance from a snapshot is much faster than adding documents manually or starting from a dump.
 
-### Version compatibility
+<Capsule intent="warning">
+For security reasons, Meilisearch will never overwrite an existing database. By default, Meilisearch will throw an error when importing a snapshot if there is any data in your instance.
 
-Since a snapshot is an exact replica of your database, it can only be opened by the same version of Meilisearch that created it.
+You can change this behavior by specifying [`--ignore-snapshot-if-db-exists=true`](/learn/configuration/instance_options#ignore-dump-if-db-exists).
```

Comment:
In my first reading, these two sentences kind of make it sound like that: by using the flag, Meilisearch will overwrite your DB with the new snapshots.
While it’s the opposite (as indicated by the name of the flag that I didn’t read because I’m a developer).

```suggestion
Instead of throwing an error, you can specify [`--ignore-snapshot-if-db-exists=true`](/learn/configuration/instance_options#ignore-dump-if-db-exists) to keep your database in place if there is one.
```